plugins/rbac/src/utils/rbac-utils.ts

```diff
@@ -5,8 +5,11 @@ import {
   stringifyEntityRef,
 } from '@backstage/catalog-model';
 
+import _ from 'lodash';
```

Comment:
I think it's better to import just the one function you're using.  And I dunno, maybe this dependency isn't needed, here's [something to consider](https://youmightnotneed.com/lodash#uniqBy).